test/unit_tests/braket/aws/test_aws_session.py

```diff
@@ -998,6 +997,11 @@ def test_upload_to_s3(aws_session):
     aws_session._s3.upload_file.assert_called_with(filename, bucket, key)
 
 
+def test_account_id_idempotency(aws_session):
+    account_id = aws_session.account_id
+    assert account_id == aws_session.account_id
```

Comment:
Do we have a test to assert the correct sts call happens the first time? We not have one test for both? ie using an aws_session mock without an account_id, and asserting the mocked sts call return value is equal to the account_id property, but only called once for multiple accesses